src/Actor/You.pm

```diff
@@ -117,6 +122,24 @@ sub inventory {
 }
 
 ##
+# InventoryList $char->storage()
+# Ensures: defined(result)
+#
+# Get the storage list for this character.
+sub storage {
+	return $_[0]->{__storage};
+}
```

Comment:
Do we really need an indirection there or it's just for writing `$char->storage->...` instead of `$char->{storage}->...`?